x-pack/plugins/security_solution/scripts/endpoint/resolver_generator_script.ts

```diff
@@ -143,6 +143,12 @@ async function main() {
       default: 'metrics-endpoint.metadata-default',
       type: 'string',
     },
+    metricsIndex: {
```

Comment:
is this needed?

The indexes above were added at teh very begining when everyone (I think) had some config in their `kibana.dev.yaml` to namespace their data on the shared SIM dev server. If we don't need to allow customization of the index, I would say remove this